.travis.yml

```diff
@@ -17,7 +17,6 @@ addons:
 
 cache:
   directories:
-  - node_modules
```

Comment:
Going to add it back, just clearing it as locally everything is installing fine :)